### PR TITLE
> Add registration tests

### PR DESCRIPTION
**Pull request by Vladyslav1992**
> Add parameters for registration tests
> Add valid registration test

### CODE REVIEW

File: SimpleAutomationCommon/Pages/RegistrationPg/RegistrationPage.cs [hunk 1]
```diff
@@ -118,6 +120,12 @@ public void Submit()
         public bool IsLoaded()
             => RegisterButton.Exists();
 
+        public string GetUser()
+        {
+            var userName = FullName.Get();
+            return userName.Remove("Hello ", "!");
+        }
```
Comment (ViacheslavMelnichenko) [inline SimpleAutomationCommon/Pages/RegistrationPg/RegistrationPage.cs]: => userName.Get().Remove("Hello ", "!");

File: SimpleAutomationCommon/Pages/AccountPage.cs [hunk 2]
```diff
@@ -18,6 +21,11 @@ public class AccountPage : Page<_>
         public IEnumerable<string> GetInfo()
             => UserInfo.Get().Remove("\r\nEdit", "\r").Split('\n').Select(i => i.Trim());
 
+        public User AddUser()
+        {
+            return new User { Email = new Email(GetInfo().Last()), FullName = GetInfo().First() };
```
Comment (ViacheslavMelnichenko) [inline SimpleAutomationCommon/Pages/AccountPage.cs]: => here

File: SimpleAutomationTests/Tests/RegistrationTests/RegistrationTests.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+            var regitrationPage = Go.To<RegistrationPage>();
+            regitrationPage.FillAndSubmit(user);
+
+            regitrationPage.IsRegistered().Should().Be(true);
+
+            var accountPage = Go.To<AccountPage>();
+            var userInfo = accountPage.AddUser();
+            userInfo.FullName.Should().Be(user.FullName, $"Because fullName should be {user.FullName}");
+            userInfo.Email.Value.Should().Be(user.Email.Value, $"Because Email should be {user.Email.Value}");
```
Comment (ViacheslavMelnichenko) [inline SimpleAutomationTests/Tests/RegistrationTests/RegistrationTests.cs]: there should be user object checking instead of fields checking!

Comment (Vladyslav1992) [reply SimpleAutomationTests/Tests/RegistrationTests/RegistrationTests.cs]: holly shit

**Pull request merged**